src/azul/indexer/aggregate.py

```diff
@@ -163,6 +163,29 @@ def get(self):
         return thaw(super().get())
 
 
+class SetOfListOfDictsAccumulator(SetOfDictAccumulator):
+    """
+    A set accumulator that supports lists of mutable mappings as values.
+    """
+
+    def __init__(self, max_size=None, key=None, filter_=None):
+        """
+        :param filter_: A callable that can be used to filter the values that
+                        are accumulated.
+        """
+        super().__init__(max_size, key)
+        self.filter_ = filter_
+
+    def accumulate(self, value) -> bool:
+        assert isinstance(value, list)
+        if self.filter_:
+            value = filter(self.filter_, value)
+        for v in value:
+            assert isinstance(v, dict)
+            status = super().accumulate(v)
+            assert status, (self.value, value)
```

Comment:
Should return status, no?

Reply:
Yeah, i think that assertion needs to be removed too.